docker-multinode/common.sh

```diff
@@ -123,40 +125,12 @@ kube::multinode::detect_lsb() {
   kube::log::status "Detected OS: ${lsb_dist}"
 }
 
-# Start a docker bootstrap for running etcd and flannel
-kube::multinode::bootstrap_daemon() {
```

Comment:
I want to keep it for both versions as the Kubernetes standard is support for two versions at the same time